Ginger/Ginger/ApplicationModelsLib/POMModels/PomElementsPage.xaml.cs

```diff
@@ -195,15 +243,48 @@ private void SetElementsGridView()
             GridViewDef view = new GridViewDef(GridViewDef.DefaultViewName);
```

Comment:
too many if's and mess - create seperate SetElementsGridView functions for regular context and for Delta view context